tox.ini

```diff
@@ -1,7 +1,14 @@
 [tox]
-envlist = py37,build_docs
+envlist = py36,py37,py37-astropydev,py36-minimal,py38,py38-numpydev,py39,build_docs
 isolated_build = True
 
+[gh-actions]
+python =
+    3.7: py37, py37-astropydev
+    3.6: py36, py36-minimal
+    3.8: py38, py38-numpydev
```

Comment:
I like how the `numpydev` and `astropydev` test runs are staggered with respect to Python versions.  That keeps the tests happening, while not making our test suite excessively long.

Reply:
Admittedly, come to think of it, we might prefer not to run those on every PR, just on master.

Reply:
Erik pointed out that that approach would delay figuring out that things break on dev versions, and if two PRs get merged between the weekly cron run, we would not be able to easily identify the offender. It's thus better to keep them here.